tsfresh/feature_extraction/feature_calculators.py

```diff
@@ -1066,12 +1066,11 @@ def time_reversal_asymmetry_statistic(x, lag):
     """
     n = len(x)
     x = np.asarray(x)
-    if 2 * lag > n:
+    if 2 * lag >= n:
         return 0
-    elif 2 * lag == n:
```

Comment:
if you have a time series of length `n`, you can only lag it `n-1`.

Here this is means, that the feature is not defined for 2 * lag == n.